pkg/services/sqlstore/stats_test.go

```diff
@@ -21,13 +21,14 @@ func TestIntegrationStatsDataAccess(t *testing.T) {
 		query := models.GetSystemStatsQuery{}
 		err := sqlStore.GetSystemStats(context.Background(), &query)
 		require.NoError(t, err)
-		assert.Equal(t, int64(3), query.Result.Users)
+		assert.Equal(t, int64(5), query.Result.Users)
```

Comment:
You'll probably need to add a dialect check here as well since you're bypassing the creation of the 2 extra users

Reply:
will move this to the loginservice! 👍